docs/google/floquet.ipynb

```diff
@@ -0,0 +1,1348 @@
+{
```

Comment:
<p>This is now make_zeta_chi_gamma_compensation_for_moments</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3765/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='84'/>

Reply:
This is still not resolved, right? Could you update this branch to Cirq master and update the method names?

Reply:
<p>Done!</p>